util/patch/patch_test.go

```diff
@@ -51,8 +51,7 @@ func TestHelperUnstructuredPatch(t *testing.T) {
 			},
 		},
 	}
-	fakeClient := fake.NewFakeClientWithScheme(scheme.Scheme)
-	g.Expect(fakeClient.Create(ctx, obj)).To(Succeed())
+	fakeClient := fake.NewFakeClientWithScheme(scheme.Scheme, obj)
```

Comment:
With controller-runtime v0.6.0 ResourceVersion behavior is different if resources are populated with `NewFakeClientWithScheme` (will not set a ResourceVersion if empty) compared to with `Create` (will set a ResourceVersion)